vent/common/values.py

```diff
@@ -24,12 +24,13 @@ class ValueName(Enum):
     ValueName.PIP,
     ValueName.PIP_TIME,
     ValueName.PEEP,
-    ValueName.PEEP_TIME,
     ValueName.BREATHS_PER_MINUTE,
     ValueName.INSPIRATION_TIME_SEC,
 }
 
 non_controllable_values = {
+    # TODO: is PEEP_TIME controllable? control_module doesn't allow set_control on it
```

Comment:
It is, but ONLY with PID control. It might throw alarms when setting a PEEP-time and using the state machine. I can enable this, though.

Reply:
Okay. Let's not deal with it in this pull request as this only involves the unit test of the coordinator.